test/distributed/fsdp/test_fsdp_optim_state.py

```diff
@@ -473,6 +473,23 @@ def test_optim_state_dict_nested(
         are incorrectly mapped to values. Their correct mapping is tested in
         other tests that exercise the save/load workflow.
         """
+        self.run_subtests(
```

Comment:
All but a few edge case tests now subtest both `use_optim_input=True` and `False` to test both code paths.

Reply:
awesome!